redbot/cogs/mod/mutes.py

```diff
@@ -322,6 +330,14 @@ async def unmute_voice(
             await ctx.send(
                 _("Unmuted {user} in channel {channel.name}").format(user=user, channel=channel)
             )
+            try:
+                await user.move_to(channel)
+            except discord.Forbidden:
+                await ctx.send(
+                    _(
+                        "Because I don't have the move members permission, this will take into effect when the user rejoins."
```

Comment:
Permission name should be title-cased same as it is presented in Discord's UI.
```suggestion
                        "Because I don't have the Move Members permission, this will take into effect when the user rejoins."
```